app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,29 @@
+class CommentsController < ApplicationController
+  before_action :find_commentable, only: :create
+
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @commentable.comments.build(comment_params)
```

Comment:
Let's rewrite it like:

``comment = @commentable.comments.build(comment_params)``
``if comment.save``
  ``redirect_to article_path(comment.article)``
``else``
  ``show error``
``end``